recipes/andreasbuhr-cppcoro/all/conanfile.py

```diff
@@ -77,6 +77,8 @@ def source(self):
     def _configure_cmake(self):
         if not self._cmake:
             self._cmake = CMake(self)
+            if self.settings.os == "Windows" and self.options.shared:
+                self._cmake.definitions["CMAKE_WINDOWS_EXPORT_ALL_SYMBOLS"] = "ON"
```

Comment:
We usually stick this in the CMake wrapper.. https://github.com/conan-io/conan-center-index/blob/8bb0aa6b8e705ee07fee9fe59e00ace63ae43d43/recipes/andreasbuhr-cppcoro/all/CMakeLists.txt#L1 then requires 3.4

Reply:
Updated